### PR TITLE
Add stdout output

### PR DESCRIPTION
**Pull request by afdesk**

### CODE REVIEW

File: cfg.yaml [hunk 1]
```diff
@@ -64,36 +72,36 @@ outputs:
...
     
 - name: my-slack
   type: slack
-  enable: true
+  enable: false
   url: https://hooks.slack.com/services/TAAAA/BBB/<key>
 
 - name: ms-team
-  type: teams
+  type: false
```
Comment (AndreyLevchenko) [inline cfg.yaml:86]: `type: false` --??

File: cfg.yaml [hunk 2]
```diff
@@ -38,9 +42,13 @@ templates:
 
 # Outputs are target services that should consume the messages
 outputs:
+- name: stdout
+  type: stdout
+  enable: true
+
 - name: my-jira   #  name must be unique
   type: jira      #  supported types: jira, email
-  enable: true
```
Comment (AndreyLevchenko) [inline cfg.yaml:43]: let's keep them enabled

File: cfg.yaml [hunk 3]
```diff
@@ -9,14 +9,18 @@ db-verify-interval: 1   #  How often to check the DB size. By default Postee che
 
 # Routes are used to define how to handle an incoming message
 routes:
-- name: route1                                 #  Route name. Must be unique
```
Comment (AndreyLevchenko) [inline cfg.yaml:12]: I would keep it as example

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: cfg.yaml (hunk 3 above)
Comment (afdesk) [reply cfg.yaml:12]: this `route` was commented as an example.

File: cfg.yaml (hunk 2 above)
Comment (afdesk) [reply cfg.yaml:43]: I've set enable fields to `false` because it's a part of output settings.

File: cfg.yaml (hunk 1 above)
Comment (afdesk) [reply cfg.yaml]: sure `teams`. fixed.

**Review by jerbia — APPROVED** [review]

**Pull request merged**